student/src/main/java/tds/student/web/handlers/TestResponseHandler.java

```diff
@@ -49,6 +50,7 @@
 
 @Controller
 @Scope ("request")
+@RequestMapping(produces = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
I'm a bit concerned with the class level request mapping forcing JSON here.  For example, https://github.com/SmarterApp/TDS_Student/pull/101/files#diff-ca388e072dbdc1dca4e729a06468d52aR110 writes XML to the response.  Does that produce json now?